plugin/slime.vim

```diff
@@ -189,7 +192,7 @@ function! s:VimterminalConfig() abort
         let cmd = &shell
       endif
       let winid = win_getid()
-      let new_bufnr = term_start(cmd)
+      let new_bufnr = term_start(cmd, b:vimterminal_config)
```

Comment:
would this be better as a new variable? `g:slime_vimterminal_config` which you could fill with anything?

You would have to handle missing or empty cases too.

Reply:
ok, I've created a new variable and handled empty cases as well. I've also updated the documentation to reflect this

Reply:
Why both a `b:` and a `g:` variables? Would there be a case where you would want buffer-specific overrides?
In practice, what would that look like?

Reply:
If you don't expect `b:`-level overrides, would this be simpler?

```
if exists("g:slime_vimterminal_config")
  let new_bufnr = term_start(cmd, g:slime_vimterminal_config)
else
  let new_bufnr = term_start(cmd)
end
```

This would go around line 195. And line 176-180 wouldn't have to be touched.

Reply:
your suggestion make sense. should I push a new commit with the correction?